lib/src/components/board/client.dart

```diff
@@ -120,6 +123,22 @@ class BoardClient extends Board implements ResourceRPCClient {
     return response.value.toInt();
   }
 
+  @override
+  Future<void> addCallbacks(List<String> interrupts, Queue<Tick> tickQueue, {Map<String, dynamic>? extra}) async {
+    // addCallbacks not implemented on client side since it is a helper for StreamTicks.
+    throw UnimplementedError();
+  }
+
+  @override
+  Stream<Tick> streamTicks(List<String> interrupts, {Map<String, dynamic>? extra}) {
+    final response = client.streamTicks(StreamTicksRequest()
+      ..name = name
+      ..pinNames.addAll(interrupts));
```

Comment:
looks like we're not passing along `extra` here, is that intentional?

Reply:
not intentional thanks for catching! added